modules/tinymce/src/plugins/table/main/ts/ui/MenuItems.ts

```diff
@@ -280,6 +280,14 @@ const addMenuItems = (editor: Editor, selectionTargets: SelectionTargets, clipbo
     ),
     onSetup: selectionTargets.onSetupCellOrRow
   });
+
+  editor.ui.registry.addMenuItem('tablecaption', {
```

Comment:
This is missing the `text` property and looking back we missed this in all your other PRs as well :facepalm: Please fix this and the other menu items (feel free to do the others in another PR). Here's what it currently looks like, so it also shows this likely wasn't tested:
![image](https://user-images.githubusercontent.com/1575550/121447782-be234200-c9d9-11eb-8282-8644d4c6d956.png)

```suggestion
  editor.ui.registry.addToggleMenuItem('tablecaption', {
    text: 'Table caption',
```

- [ ] Fix missing `text` field on menu items
- [ ] Ensure tests exist for this menu item

Reply:
Additionally, this needs to be a toggle menu item otherwise the toggling won't work.

- [ ] Fix incorrect registration of toggle menu item

Reply:
If there's any new strings for the `text` properties, we'll need to make a note for translations too. I don't know if we have strings for these things yet, so thought I'd mention.

Reply:
There should not be as they are already being used in the tooltips for buttons. The only case that should apply to is if we added new strings for both.